### ISSUE TITLE
txt2img and Img2img stuck at 0% if ControlNet is enabled,how to fix it?

### ISSUE THREAD

**Issue by wuyue0509** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#247]
No response in SD's console

**Comment by fractalfantasy** [comment]
same here!

**Comment by wuyue0509** [comment]
> 同样在这里！

I already fixed it,you must go back controlnet v1.1.134 version

**Comment by 501ZHY** [comment]
Same

**Comment by 501ZHY** [comment]
no console output

**Comment by 501ZHY** [comment]
> > 同样在这里！
> 
> I already fixed it,you must go back controlnet v1.1.134 version

How do you roll back controlnet  version? are u using git?

**Comment by gee666** [comment]
same here, stuck at 0%, nothing's happening, no errors in console

**Comment by 501ZHY** [comment]
This issue is caused by the api change of controlnet plugin. For those who had this issue. you can open your cmd tool in the stable-diffusion-webui-controlnet folder by using "cd /d "command. Then roll back the controlnet version to v1.1.134 by using "git reset --hard 5d387ab"

**Comment by 501ZHY** [comment]
> 

This issue is caused by the api change of controlnet plugin. For those who had this issue. you can open your cmd tool in the stable-diffusion-webui-controlnet folder by using "cd /d "command. Then roll back the controlnet version to v1.1.134 by using "git reset --hard 5d387ab"

**Comment by 501ZHY** [comment]
> same here, stuck at 0%, nothing's happening, no errors in console

This issue is caused by the api change of controlnet plugin. For those who had this issue. you can open your cmd tool in the stable-diffusion-webui-controlnet folder by using "cd /d "command. Then roll back the controlnet version to v1.1.134 by using "git reset --hard 5d387ab"

**Comment by AbdullahAlfaraj** [comment]
@wuyue0509 
can you provide me with more info?
- installation method
- screenshot of the plugin

also are you using controlnet version c9c8ca6e?

**Comment by gee666** [comment]
> update your plugin, this has been resolved in v1.2.3

except that 1.2.3 doesn't generate anything at all with or without ControlNet. Waiting very much for another fix.

**Comment by wuyue0509** [comment]
> @wuyue0509 你能给我提供更多信息吗？
> 
> * 安装方法
> * 插件截图
> 
> 您还在使用 controlnet 版本 c9c8ca6e 吗？

yes,it does not work at all,please check it out and fix it,thx a lot.
